.github/workflows/generate-changelog/action.yml

```diff
@@ -23,3 +27,30 @@ runs:
       run: |
             npm install --no-package-lock --no-save @octokit/core@3.4.0
             node ./.github/scripts/generate-changelog.js
+
+            # Upload-artifact action fails if the file size is 0.
+            if [[ ${{ inputs.GENERATE_EMPTY_CHANGELOG }} == "true" ]]; then
+              # Check the file size
+              size=$(wc -c < "${{ env.CHANGELOG_FILE }}")
+              if [ $size -eq 0 ]; then
+                echo "No changelog" > "${{ env.CHANGELOG_FILE }}"
+              fi
+            fi
+
+            CHANGELOG_CONTENT=$(cat ${{ env.CHANGELOG_FILE }} )
+
+            # Replace ED-xxxx with a link to the corresponding Jira issue
+            CHANGELOG_CONTENT=$(echo "$CHANGELOG_CONTENT" | sed -E 's/\[ED-([0-9]+)\]/<https:\/\/elementor.atlassian.net\/browse\/ED-\1|\*ED-\1*\>/g')
```

Comment:
I'm not sure we should do this - people from outside Elementor do not have access to our Jira - lets skip it for now
```suggestion
```

Reply:
Done